azext_iot/pnp/providers/__init__.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.sdk.pnp.modelrepository import ModelRepositoryControlPlaneApi
+from azext_iot.sdk.pnp.dataplane import DigitalTwinModelRepositoryApi
+from azext_iot.digitaltwins.providers.auth import DigitalTwinAuthentication
+from azext_iot.constants import PNP_ENDPOINT, PNP_TENANT_RESOURCE_ID
+from msrestazure.azure_exceptions import CloudError
+
+__all__ = [
+    "pnp_modelrepo_service_factory",
+    "pnp_digitaltwin_modelrepo_api_service_factory",
+    "PnPModelRepositoryApiManager",
+    "PnPModelRepositoryManager",
+    "CloudError",
+]
+
+
+def pnp_modelrepo_service_factory(cmd, pnp_dns_suffix=None, *_):
```

Comment:
nit - why not `pnp_dns_suffix=PNP_ENDPOINT`

Reply:
This only gets called on instantiation of the sdk - in which we have to pass `None` or the new suffix to the call.

The default argument won't get set if it's `None`, since the previous method `get_mgmt_sdk` will always resolve it to None regardless, so we'll still need the `if pnp_dns_suffix else default_val`